src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Stars</title>
     <link
```

Comment:
The `link` tag for the stylesheet should be formatted with each attribute starting on a new line with a 2-space indentation relative to the tag to keep your attributes correctly formatted.